.github/workflows/ci.yml

```diff
@@ -77,3 +77,51 @@ jobs:
           git status --porcelain
           git diff --no-ext-diff --quiet --exit-code
         name: Check not out of date generated files
+
+  build-dev:
+    name: Build And Test (-dev.x)
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - run: npm install -g pnpm@7.27.1
+        name: Install PNPM
+
+      # Install deps and build the bump package first
+      # - run: pnpm install
+      #   name: Install dependencies
+
+      # - run: npm run build
+      #   name: Build
+      #   working-directory: ./typespec-bump-deps
```

Comment:
Remove this?

Reply:
ah yeah, had issue originally because the next command kept trying to load the unbuilt version of the typespec-bump-deps first